doc/modules/extractors.rst

```diff
@@ -27,40 +28,52 @@ Every format can be read with a simple function:
 
 .. code-block:: python
 
-    recording_oe = read_openephys("open-ephys-folder")
+    recording_oe = read_openephys(folder_path="open-ephys-folder")
 
-    recording_spikeglx = read_spikeglx("spikeglx-folder")
+    recording_spikeglx = read_spikeglx(folder_path="spikeglx-folder")
 
-    recording_blackrock = read_blackrock("blackrock-folder")
+    recording_blackrock = read_blackrock(folder_path="blackrock-folder")
 
-    recording_mearec = read_mearec("mearec_file.h5")
+    recording_mearec = read_mearec(file_path="mearec_file.h5")
 
 
 Importantly, some formats directly handle the probe information:
 
 .. code-block:: python
 
-    recording_spikeglx = read_spikeglx("spikeglx-folder")
+    recording_spikeglx = read_spikeglx(folder_path="spikeglx-folder")
     print(recording_spikeglx.get_probe())
 
-    recording_mearec = read_mearec("mearec_file.h5")
+    recording_mearec = read_mearec(file_path="mearec_file.h5")
     print(recording_mearec.get_probe())
 
 
+Although most recordings are loaded with the :py:mod:`~spikeinterface.extractors`
+a few file formats are loaded from the :py:mod:`~spikeinterface.core` module
+
+.. code-block:: python
+
+    import spikeinterface as si
+
+    recording_binary = si.read_binary(file_path='binary.bin')
+
+    recording_zarr = si.read_zarr(file_path='zarr_file.zarr')
```

Comment:
@zm711 note that recently I exposed the `read_binary` and `read_zarr` functions to the extractors mofule too. (https://github.com/SpikeInterface/spikeinterface/blob/main/src/spikeinterface/extractors/extractorlist.py#L14-L15)

I think we can remove this paragrapth for sake of clarity, i.e., all `read_*` functions are in `extractors`. What do you think?

Reply:
I didn't realize that (Thanks :)). The autofunction is connected to the source, so I thought maybe it was in core. I'll delete (it is better if they are all extractor).